MastermindLogic/src/test/java/mastermindlogictests/MastermindLogicApplicationTests.java

```diff
@@ -18,8 +16,8 @@
 	void setup() {
 		board1 = new Board(1);
 		Board board2 = new Board(2);
-		player1 = new Player(1, "JohnDoe", "secret", board1);
-		player2 = new Player(2, "JaneDoe", "secret", board2);
+		player1 = new Player(1);
```

Comment:
namen mogen wel nog geset worden maar wel met een setName() en dergelijke (is er mogelijk nog niet, want dit is nog niet gemaakt. Die mag je wel aanmaken)